docs/resources/cce_node_pool_v3.md

```diff
@@ -46,14 +46,16 @@ The following arguments are supported:
 
 * `flavor` - (Required) Specifies the flavor id. Changing this parameter will create a new resource.
 
-* `availability_zone` - (Required) specify the name of the available partition (AZ). Changing this parameter will create a new resource.
+* `availability_zone` - (Required) Specify the name of the available partition (AZ). If zone is not
+  specified than `node_pool` will be across `random`. Changing this parameter will create a new resource.
```

Comment:
"Across `random`" what? I mean, maybe it's better to put it like "will be in randomly selected AZ"